### DISCUSSION TITLE
Chat triggers

### DISCUSSION THREAD

**Discussion by bionictoothpick** [gottagofaster236/RewardsTheater#3] [Ideas]
Thank you for sharing this plugin.

This is a feature request. I'd like users to be able to have an intro sound or meme if they would like that isn't really a channel reward from points, but more like a trigger...so if a specific level user were to type: !gottagofaster236 it would play a media source.

Thoughts?

**Comment by gottagofaster236** [comment]
> Thank you for sharing this plugin.
> 
> This is a feature request. I'd like users to be able to have an intro sound or meme if they would like that isn't really a channel reward from points, but more like a trigger...so if a specific level user were to type: !gottagofaster236 it would play a media source.
> 
> Thoughts?

Hi, thanks for the idea. It probably shouldn't be too difficult to implement, although this is kind of orthogonal to what this plugin does.
Do you have any ideas as to how this could be implemented in UI, though? I don't really want to overcomplicate it 😅

**Reply by bionictoothpick** [reply]
If your plugin sees chat and the role of the chatter the interface could be something like:

ChatMessage   UserRole             Action
!lurking            moderator     PlayMediaSource
!entrysong       any chatter    PlayMediaSource

**Reply by bionictoothpick** [reply]
I have an easier explanation...when someone redeems a reward, could that reward just be to enable a source?

**Reply by gottagofaster236** [reply]
Hi again, sorry for not responding earlier. Tbh I'd rather keep the plugin focused on one thing (videos for channel points). If you'd like to set up more complicated things, like chat triggers, I think I'd be easier to just use a more comprehensive program such as [streamer.bot](https://streamer.bot/). Cheers

**Reply by bionictoothpick** [reply]
Thanks for the quick reply. I appreciate your work. Have a great 2024.
